Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -887,11 +890,13 @@ - (void)validateActiveTargets {
     XCTAssertNotEqual(found, actualTargets.end(), @"Expected active target not found: %s",
                       targetData.ToString().c_str());
 
-    // TODO(mcg): validate the purpose of the target once it's possible to encode that in the
-    // spec tests. For now, only validate properties that can be validated.
+    // TODO(Mila): Replace the XCTAssertEqual() checks on the individual properties of TargetData
```

Comment:
nit: Remove your name from this "TODO" comment. By putting your name there it implies that you are actively keeping track of this work and will do it when the time comes. That is _not_ the case, so just leave it as an "unassigned" TODO to record that _somebody_ should do this in the future.

Reply:
`// TODO:...` gives error messages like this: 
<img width="1280" alt="Screenshot 2023-03-16 at 9 22 12 AM" src="https://user-images.githubusercontent.com/107142260/225685946-79c121da-6f68-4a2b-853a-27ebde5962c4.png">